mldsa/fips202/fips202.h

```diff
@@ -75,13 +111,28 @@ __contract__(
 );
 
 #define shake128_release FIPS202_NAMESPACE(shake128_release)
+/*************************************************
+ * Name:        shake128_release
+ *
+ * Description: Release and securely zero the SHAKE128 state.
+ *
+ * Arguments:   - mld_shake128ctx *state: pointer to Keccak state
+ **************************************************/
 void shake128_release(mld_shake128ctx *state)
 __contract__(
   requires(memory_no_alias(state, sizeof(mld_shake128ctx)))
   assigns(memory_slice(state, sizeof(mld_shake128ctx)))
 );
 
 #define shake256_init FIPS202_NAMESPACE(shake256_init)
+/*************************************************
+ * Name:        shake256_init
+ *
+ * Description: Initializes Keccak state for use as SHAKE256 XOF
```

Comment:
nit: The mentioning of `Keccak` feels misplaced here now that the context is generic.